man/systemd-analyze.xml

```diff
@@ -1183,6 +1189,14 @@ boot  efi  exitrd  init  lib32	libx32	media	    opt  root  sbin  sys  usr  vmlin
 
       <xi:include href="version-info.xml" xpointer="v258"/>
     </refsect2>
+
+    <refsect2>
+      <title><command>systemd-analyze unit-gdb <replaceable>SERVICE</replaceable></command></title>
+
+      <para>Spawn and attach a debugger to the given service. If no debugger is specified, defaults to gdb.</para>
```

Comment:
Please mention that debugger can be specified with `--debugger=` option or `$SYSTEMD_DEBUGGER` environment variable. See coredumpctl(1).

Also, make `gdb` as a reference to gdb(1).

See man/coredumpctl.xml.